www/docs/main/example-apps.mdx

```diff
@@ -0,0 +1,183 @@
+
+```bash
+npx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+
+```sh
+yarn create next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```sh
+pnpx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
+```
+
+  </TabItem>
+</Tabs>
+
+- Example deploy: [nextjs.trpc.io](https://nextjs.trpc.io)
+- [Source code](https://github.com/trpc/examples-next-prisma-starter)
+
+---
+
+### zART-stack example (zero-API, React, TypeScript)
+
+<Tabs>
+  <TabItem value="git clone" label="git clone">
+
+```bash
+git clone git@github.com:KATT/zart.git
+```
+
+  </TabItem>
+</Tabs>
+
+- [Source code](https://github.com/KATT/zART)
+
+---
+
+### Next.js starter with Prisma, E2E testing, ESLint, next-auth, WebSockets, and subscriptions
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```bash
+npx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter-websockets trpc-prisma-starter-websockets
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+
+```sh
+yarn create next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter-websockets trpc-prisma-starter-websockets
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```sh
+pnpx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter-websockets trpc-prisma-starter-websockets
+```
+
+  </TabItem>
+</Tabs>
+
+- Example deploy: [websockets.trpc.io](https://websockets.trpc.io)
+- [CodeSandbox](https://codesandbox.io/s/github/trpc/trpc/tree/main/examples/next-prisma-starter-websockets?file=/src/pages/index.tsx)
+- [Source code](https://github.com/trpc/examples-next-prisma-starter-websockets)
+
+---
+
+### Next.js Todo MVC example with SSG and Prisma
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```bash
+npx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-todomvc trpc-todo
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+
+```sh
+yarn create next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-todomvc trpc-todo
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```sh
+npx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-todomvc trpc-todo
+
+```
+
+  </TabItem>
+</Tabs>
+
+- Example deploy: [todomvc.trpc.io](https://todomvc.trpc.io)
+- [CodeSandbox](https://codesandbox.io/s/github/trpc/trpc/tree/main/examples/next-prisma-todomvc?file=/pages/%5Bfilter%5D.tsx)
+- [Source code](https://github.com/trpc/examples-next-prisma-todomvc)
+
+---
+
+### Express server with tRPC Shield and Prisma
+
+<Tabs>
```

Comment:
Any reason you didn't include yarn/pnpm tabs here?

Reply:
I wasn't sure if `create-trpc-appx` was going to work with `yarn` and `pnpm` so I left it to just npx. I can add if they will!